### PR TITLE
Update event API endpoint tests

### LINKED ISSUE #167: Update /api/event tests based on updated endpoints

### PR DESCRIPTION
**Pull request by kevhuang**
Resolves #167 

@stevenshuhyow I made changes based on the changes you made to the events API endpoints. Let me know if I missed or fudged anything.

### CODE REVIEW

File: client/public/js/utils/EventAPI.js [hunk 1]
```diff
@@ -4,24 +4,6 @@
 
 var EventAPI = {
   /**
```
Comment (ekong2) [inline client/public/js/utils/EventAPI.js:6]: thanks for removing this huge block

File: server/api/event/index.js [hunk 2]
```diff
@@ -1,17 +1,6 @@
 var router = require('express').Router();
 var db = require('../../db');
```
Comment (ekong2) [inline server/api/event/index.js:3]: this conflicts with derek's, we can go ahead and merge yours in first and let him rebase

**Comment by ekong2** [comment]
initial testing yielded 17 passing, 16 failed. want to review the failing tests together?

**Comment by ekong2** [comment]
also seems like the travis build seems to error out when a connection to a database hasn't been established yet

**Comment by dmsakamoto** [comment]
:+1:

**Pull request merged**